src/main/java/mate/academy/service/MovieSessionService.java

```diff
@@ -5,9 +5,13 @@
 import mate.academy.model.MovieSession;
 
 public interface MovieSessionService {
-    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+    MovieSession add(MovieSession movieSession);
 
     MovieSession get(Long id);
 
-    MovieSession add(MovieSession session);
+    List<MovieSession> findAvailableSessions(Long movieId, LocalDate date);
+
+    Boolean update(MovieSession movieSession);
+
+    Boolean delete(Long id);
```

Comment:
Consider changing the return type of the `delete` method to `void` to better reflect the operation's outcome. Returning a `Boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.